src/test/java/InfrastructureManager/NetworkStructure/NetworkTest.java

```diff
@@ -0,0 +1,49 @@
+package InfrastructureManager.NetworkStructure;
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.Arrays;
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+/**
+ * This class contains test cases for Network Structure Module.
+ *
+ * @author Shankar Lokeshwara
+ */
+public class NetworkTest {
+	
+	/**
+	 * Test case to write JSON string
+	 */
+	@Test
+	public void saveNetworkTest() throws JsonProcessingException {
+	 // create network object
+    	Network network = new Network();
+    	network.setDeviceList(Arrays.asList(new Device(001),new Device(002),new Device(003),new Device(004)));
+    	network.setConnectionList(Arrays.asList(new Connection(100),new Connection(101),new Connection(102),new Connection(103),new Connection(104)));
+    	network.setLocationList(Arrays.asList(new Location(500),new Location(501),new Location(502),new Location(503),new Location(504)));
+    	network.setApplicationList(Arrays.asList(new ApplicationInstance(600),new ApplicationInstance(601),new ApplicationInstance(602),new ApplicationInstance(603),new ApplicationInstance(604)));
+		String writeString = network.saveNetwork();
+		System.out.println(writeString);
+		int deviceID0 = network.getDeviceList().get(0).getDeviceId();
+		System.out.println(deviceID0);
+	    Assert.assertEquals(1, deviceID0);
```

Comment:
you only tested the first device id value, what about all of the others values that you assigned?
and what about writeString?
also, you are mixing between testing the setters and testing saveNetwork, each should has its own test.